### PR TITLE
convolution2d

### PR DESCRIPTION
**Pull request by alrojo**
explicit use "is not None" for activation_fn, normalize_fn and provide more detailed description

### CODE REVIEW

**Comment by danmane** [comment]
@sguada PTAL

File: tensorflow/contrib/layers/python/layers/layers.py [hunk 1]
```diff
@@ -379,9 +379,11 @@ def convolution2d(inputs,
       If greater than 1, than the a'trous convolution is applied and `stride`
       must be set to 1.
     activation_fn: activation function.
+      set to None for linear activation
```
Comment (sguada) [inline tensorflow/contrib/layers/python/layers/layers.py]: Move to the line above.

Comment (sguada) [inline tensorflow/contrib/layers/python/layers/layers.py]: rewrite as
activation_fn: activation function, set to None to skip it and maintain a linear activation.

**Comment by sguada** [comment]
LGTM,
jenkins, test this please

File: tensorflow/contrib/layers/python/layers/layers.py (hunk 1 above)
Comment (alrojo) [inline tensorflow/contrib/layers/python/layers/layers.py]: @sguada Done

File: tensorflow/contrib/layers/python/layers/layers.py [hunk 2]
```diff
@@ -378,8 +378,7 @@ def convolution2d(inputs,
     rate: integer. If less than or equal to 1, a standard convolution is used.
       If greater than 1, than the a'trous convolution is applied and `stride`
       must be set to 1.
-    activation_fn: activation function.
-      set to None for linear activation
+    activation_fn: activation function, set to None to skip it and maintain a linear activation.
```
Comment (sguada) [inline tensorflow/contrib/layers/python/layers/layers.py]: Sorry try to keep the max length of each line under 80 chars.
activation_fn: activation function, set to None to skip it and maintain
  a linear activation.

**Comment by sguada** [comment]
@danmane do you know why Jenkins is not running?

File: tensorflow/contrib/layers/python/layers/layers.py (hunk 2 above)
Comment (alrojo) [inline tensorflow/contrib/layers/python/layers/layers.py]: @sguada updated

**Comment by sguada** [comment]
@alrojo would you mind updating the other layers that have activation_fn and normalizer_fn too?

**Comment by sguada** [comment]
@tensorflow-jenkins test this please

**Comment by alrojo** [comment]
@sguada I'll look into the other layers, why did the test fail?

**Comment by danmane** [comment]
It looks like the test failure was a flake, since it was a timeout in code clearly unrelated to your change. I would consider that a clean build, but I will re-run the tests anyway.
@tensorflow-jenkins test this please.

**Comment by alrojo** [comment]
@sguada updated all of the other layers with activation_fn and normalizer_fn. I also updated a typo in the docs: normalize_fn -> normalizer_fn

**Comment by sguada** [comment]
Thanks @alrojo 

@tensorflow-jenkins test this please

**Comment by martinwicke** [comment]
Merged, the remaining test failure was a flake.

**Pull request merged**